mentorship/gsod/2022/project-ideas.md

```diff
@@ -5,23 +5,104 @@ which Keptn contributors have proposed for this year.
+
+For this deliverable, instructions should be pedagogical rather than a pure cookbook:
+provide enough explanation that the user gains some understanding of what they are doing.
+Include information for verifying each step and some troubleshooting information or links.
+
+#### New example projects
+
+Create 5-6 projects that introduce key Keptn capabilities for the Keptn curious to explore. 
+We have a number of existing tutorials to use as starters or inspiration. 
+Note that many of the existing tutorials use the [sockshop project](https://github.com/keptn-demo/sockshop) as a base;
+and some updates will be needed there.
+
+Each project should include:
+
+- Files that can be downloaded from GitHub to provide some basic configuration with an explanation of what is provided in these files (that they would have to do themselves if they wanted to implement this functionality.
+- The specification in GitHub for what needs to be installed to provide a viable environment for the project, including integrations that must be included, configurations that must be set, and so forth.
+- Excellent information about how to run the sample project and do the exercises, including context for each step, what to look for to determine success, and understanding the output.  The goal is to begin to educate the user about how to get the most out of Keptn
+
+#### Separate guides for roles
+
+We want to introduce separate documentation for different Keptn user roles: 
+administrators, end-users, and developers.
+We suggest to work together to regroup the content so that the new structure is introduced.
+
+#### New reference pages
+
+Rewrite reference pages for files to conform to the structure proposed in the [Structure of reference pages](https://docs.google.com/document/d/1rmSGUFsnYuNc_BGHGmkW-9iB3hrOYbMeA-mJmMOjGRk/edit?usp=sharing) proposal.
+We are currently creating reference pages for critical files by capturing information in existing doc files, source code README files, etc.
+This project would be to restructure and expand the information in these files to create proper reference pages.
+It might also include relocating some information in existing files to a more appropriate location.
+
+Same for other standard documentation independent from the version.
+Set a standard for what information belongs in a “Concepts” article versus
+what belongs in reference pages or guides that are currently
+located in the version-specific sections of the documentation.
+Then modify the existing Concepts articles to conform to this standard and create Concepts articles for appropriate concepts that
+are discussed in existing guides or elsewhere
+(such as README files in the software repos or existing Keptn videos).
+
+Creating missing guides is not part of this project
 
 ## Alternative project ideas
 
+We also discussed a number of alternative project ideas that can be considered.
```

Comment:
```suggestion
We also discussed several alternative project ideas that can be considered.
```

Reply:
Could it just be "We also discussed alternative project ideas..." or even just "We discussed other project ideas..."

Reply:
All well! This is also good for me :)